dushimeemma/fs-simba-cal
# ESL-1 Implement authentication functionality

Description

Implement auth functionality

Type of change

[ ] Bug fix (Non breaking change that fixes an existing issue)

[x] New Feature (Non breaking change which adds funcitonality)

[ ] Breaking change (New feature that removes or causes an existing feature not to work as expected)

[ ] This change requires a documentation update

How Has This Been Tested?

[x] Unit tests

[x] Component Tests

[ ] Integration Tests

How Should This Be Manually tested:

- Clone this repo `git clone https://github.com/dushimeemma/fs-simba-cal/`.

- Checkout the branch `git checkout feature/ESL-1-authentication`

- Run `yarn install`

- Create `.env` file in the root directory `touch .env`

- Copy everything from `.env.example` to `.env` and use real data `mv .env.example .env`

- Run `yarn dev`

Checklist:
[x] My code follows all the styles and conventions outlined for this project

[x] I have linted my code before submission

[x] I have made my corresponding changes to the documentation

[x] My changes generates no new warning

[x] I have added tests that prove my feature is working or my fix is effective

[x] New and existing tests pass locally with my changes

JIRA
[ESL-1](https://www.notion.so/Setup-NextJS-project-cbf21d0431c64e9992ce10c12f2da5d9)

Screenshot

![login](https://user-images.githubusercontent.com/64155652/149267067-37716dcd-cc7f-4e1e-98d8-18704a38f47b.png)
![signup](https://user-images.githubusercontent.com/64155652/149267095-b74bd770-c8d9-4131-914f-e1c0c4439420.png)
